firefox-ios/Client/Frontend/Home/PrivateHome/PrivateHomepageViewController.swift

```diff
@@ -130,8 +130,21 @@ final class PrivateHomepageViewController: UIViewController,
     }
 
     deinit {
-        // TODO: FXIOS-11187 - Investigate further on privateMessageCardCell memory leaking during viewing private tab.
-        scrollView.removeFromSuperview()
+        // TODO: FXIOS-13097 This is a work around until we can leverage isolated deinits
```

Comment:
Oupsy, lll revert my change in https://github.com/mozilla-mobile/firefox-ios/pull/30681 as we duplicated this fix. We can merge yours